src/Andre/SoulsFormats/SoulsFormats/Formats/MSB/MSB3/PartsParam.cs

```diff
@@ -1217,7 +1217,6 @@ public class Collision : Part
             {
                 /// <summary>
                 /// HitFilterType
-                /// Unused as it is not exhaustive
```

Comment:
Was every map checked?
I'd rather have a byte value with a decorator than an incomplete enum, so please confirm every map was checked.

Reply:
Yes, the Map Validation test loads all MSBs, if this enum lacks a value used it would throw. Obviously, there might be potential values never used, but that doesn't matter too much